<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_Lidar</class>
 <widget class="QMainWindow" name="ihm_Lidar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Graph</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="cb_active_graph">
          <property name="text">
           <string>Active graph</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Filtrage points</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="filtrage_distance">
          <property name="toolTip">
           <string>N'affiche pas les points de distance supérieurs</string>
          </property>
          <property name="suffix">
           <string> [mm]</string>
          </property>
          <property name="maximum">
           <number>4000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="3" colspan="5">
         <widget class="QCustomPlot" name="customPlot" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Zoom</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="zoom_distance">
          <property name="suffix">
           <string> [mm]</string>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="value">
           <number>3700</number>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Données affichées</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="type_affichage_graph">
          <item>
           <property name="text">
            <string>Polaire</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Linéaire</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Type d'affichage</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="data_affichees">
          <item>
           <property name="text">
            <string>Brutes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Filtrées</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QSpinBox" name="test_spin">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_sick">
       <attribute name="title">
        <string>Sick Tim5xx</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <widget class="QSpinBox" name="tim5xx_port">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>99999999</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port </string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="tim5xx_openport">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="tim5xx_closeport">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>114</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="tim5xx_ip">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>IP</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="tim5xx_enable_autoreconnect">
            <property name="text">
             <string>Enable autoreconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Sample period</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="lidar_sample_period">
          <property name="suffix">
           <string> [msec]</string>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_logger">
       <attribute name="title">
        <string>Logger</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="3">
         <widget class="QLineEdit" name="logger_pathfilename">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Nom du fichier de log</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="PB_logger_select_file">
          <property name="toolTip">
           <string>Choix du fichier signal</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="logger_sample_period_label">
          <property name="text">
           <string>Période</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="PB_start_logger">
          <property name="toolTip">
           <string>Commence la génération du signal</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/icons/player_play.png</normaloff>:/icons/player_play.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="PB_stop_logger">
          <property name="toolTip">
           <string>Arrête la génération du signal</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../icons.qrc">
            <normaloff>:/icons/player_stop.png</normaloff>:/icons/player_stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>212</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>103</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="logger_sample_period">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="specialValueText">
           <string>Tous les messages</string>
          </property>
          <property name="suffix">
           <string> [msec]</string>
          </property>
          <property name="maximum">
           <number>9000000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_replay">
       <attribute name="title">
        <string>Replay</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Paramètrage</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0" colspan="5">
            <widget class="QLineEdit" name="trace_name">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Chox du fichier de trace</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QToolButton" name="PB_choixTrace">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Choix du fichier signal</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="dureeStepTrace_label">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Durée du step</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="4">
            <widget class="QSpinBox" name="dureeStepTrace">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Vitesse de l'animation. Valeur spéciale &quot;-1&quot; pour indiquer que la durée du step est prise du fichier de trace</string>
             </property>
             <property name="specialValueText">
              <string>Fichier trace</string>
             </property>
             <property name="suffix">
              <string> [msec]</string>
             </property>
             <property name="minimum">
              <number>-1</number>
             </property>
             <property name="maximum">
              <number>9000000</number>
             </property>
             <property name="value">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="nombreCycles_label_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Nombre de cycles</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="4">
            <widget class="QSpinBox" name="nombreCyclesTrace">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Valeur spéciale &quot;-1&quot; pour indiquer que le signal est joué en boucle</string>
             </property>
             <property name="specialValueText">
              <string>En boucle</string>
             </property>
             <property name="suffix">
              <string/>
             </property>
             <property name="minimum">
              <number>-1</number>
             </property>
             <property name="maximum">
              <number>9000000</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="PB_StepBackwardTrace">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Joue le pas précédent</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/backward.png</normaloff>:/icons/backward.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QPushButton" name="PB_StartTrace">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Commence la trace</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/player_play.png</normaloff>:/icons/player_play.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="PB_StopTrace">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Arrêt de jouer la trace</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/player_stop.png</normaloff>:/icons/player_stop.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="4" colspan="2">
            <widget class="QPushButton" name="PB_StepForwardTrace">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Joue le pas suivant</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../icons.qrc">
               <normaloff>:/icons/forward.png</normaloff>:/icons/forward.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="6">
            <widget class="QSlider" name="sliderStepNumTrace">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="lbl_valStepNumTrace">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>N° ligne</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="4">
            <widget class="QLabel" name="valStepNumTrace">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
